lib/logstash/runner.rb

```diff
@@ -63,15 +44,6 @@ def run(args)
         end
         return LogStash::Agent.run($0, agent_args)
       end,
-      "rspec" => lambda do
```

Comment:
I would leave the rspec subcommand, it can help us debug user issues, see @jordansissel's comment on https://github.com/elastic/logstash/issues/2776#issuecomment-77572492, IIRC we are not shipping the `/bin/rspec` in any packages.